src/message-page.scss

```diff
@@ -0,0 +1,88 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-message-page: 0 !default;
+
+$block: #{$fd-namespace}-message-page;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-flex-center();
+
+  width: 100%;
+  height: 100%;
+  padding: 1rem;
+  background: var(--sapBackgroundColor);
+
+  &__container {
+    @include fd-reset();
+
+    @include fd-flex(column) {
+      align-items: center;
+    }
+
+    margin: auto;
+    max-width: 30rem;
+    transform: translateY(-1rem);
+  }
+
+  &__icon-container {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    width: 100%;
+    height: 8rem;
+  }
+
+  &__icon {
+    font-size: 6rem;
+    color: var(--sapContent_NonInteractiveIconColor);
+    opacity: 50%;
+  }
```

Comment:
Could you make stronger selector, since it's going to be used in with icons, that have reset mixin?

Reply:
Done